packages/wonder-blocks-tooltip/package.json

```diff
@@ -15,12 +15,12 @@
   "author": "",
   "license": "MIT",
   "dependencies": {
-    "react-popper": "^1.0.0-beta.6",
     "@khanacademy/wonder-blocks-color": "^1.0.2",
     "@khanacademy/wonder-blocks-core": "^1.0.2",
     "@khanacademy/wonder-blocks-layout": "^0.0.2",
     "@khanacademy/wonder-blocks-modal": "^0.0.3",
     "@khanacademy/wonder-blocks-spacing": "^2.0.0",
-    "@khanacademy/wonder-blocks-typography": "^1.0.2"
+    "@khanacademy/wonder-blocks-typography": "^1.0.2",
+    "react-popper": "^1.0.0"
```

Comment:
We'll have to install this in `webapp`.  When `npm` installs it is there a single bundle we can load?  If so, maybe we can install in `third_party/node_modules` and have `javascript/node_modules/react-popper/index.js` point at that bundle.